Content.Server/GameObjects/Components/Chemistry/ChemMasterComponent.cs

```diff
@@ -257,14 +256,14 @@ private void TransferReagent(string id, ReagentUnit amount, bool isBuffer)
                         ReagentUnit actualAmount;
                         if (amount == ReagentUnit.New(-1))
```

Comment:
I know it's not from this PR, but this was my first encounter with ReagentUnit and I had a hard time understanding the internal representation (and I know I'm not the only one), could you add a small comment at the top of ReagentUnit explaining what_value and Shift are?

I believe it should be something like
```
// To avoid issues with precission, _value holds the fractional and integer parts of the quantity together in a single int.
// The first 2 digits (due to Shift = 2) hold the fractional part, while the remaining digits hold the integer part. So if _value is 123, that represents an actual quantity of 1.23.
```